source/_cookbook/SSL guide.md

```diff
@@ -0,0 +1,570 @@
+---
+layout: page
+title: "Remote Access with SSL via LetsEncrypt"
+description: "A guide to remotely accessing HA and securing the connection with an SSL certificate from LetsEncrypt"
+date: 2017-03-16 17:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Infrastructure
+---
+
+This guide was added by mf_social on 16/03/2017 and was valid at the time of writing.  This guide makes the following assumptions:
+
+ * You can access your HA instance across your local network, and access the device that it is on via SSH from your local network.
+ * You know the internal IP address of your router and can access your router's configuration pages.
+ * You have already set up a password for your HA instance, following the advice on this page: [http](https://home-assistant.io/docs/configuration/basic/)

```

Comment:
The fact that a password should be set up BEFORE opening the router's firewall cannot be stressed enough.
I think there should be a fat warning note on the beginning of the page.